py/torch_tensorrt/dynamo/lowering/_decompositions.py

```diff
@@ -243,6 +243,44 @@ def empty_strided_decomposition(*args, **kwargs) -> torch.Tensor:
     )
 
 
+@register_torch_trt_decomposition(
+    torch.ops.aten.scatter_add.default, registry=TORCH_TRT_DECOMPOSITIONS
+)
+def scatter_add_decomposition(
+    input_tensor: torch.Tensor,
+    src_tensor: torch.Tensor,
+    dim: int,
+    index: torch.Tensor,
+) -> torch.Tensor:
```

Comment:
It seems the order of args doesn't match with tests, which made CI failed.

Reply:
Thanks for pointing this out! Corrected.